app/interactors/sign_up_validator.js

```diff
@@ -0,0 +1,26 @@
+const errors = require('../errors');
+
+const PASSWORD_MIN_LENGTH = 8;
+
+const validateEmail = email => {
+  const re = /@wolox\.(com|com\.cl|co|com\.mx|com\.ar)$/;
+  return re.test(email);
+};
+
+const validatePassword = password => {
+  const letter = /[a-zA-Z]/;
+  const number = /[0-9]/;
+  return number.test(password) && letter.test(password) && password.length > PASSWORD_MIN_LENGTH;
+};
+
+module.exports.validate = body => {
```

Comment:
This file isn't really an interactor either, but keep it like this for now, we'll refactor it later anyways.